### PR TITLE
[IMP] web_editor, website: add database ID to OLG calls

### PR DESCRIPTION
**Pull request by Guillaume-gdi**
Related IAP PR: https://github.com/odoo/iap-apps/pull/758
This commit adds the database ID to the IAP calls made to generate text.
It permits to prevent abuses of OpenAI calls.

task-3740440

### CODE REVIEW

File: addons/web_editor/controllers/main.py
Comment (rdeodoo) [inline addons/web_editor/controllers/main.py:812]: Can you add the term in .pot?

Comment (Guillaume-gdi) [reply addons/web_editor/controllers/main.py:812]: arff yes indeed I forgot that. It's done :+1:

Comment (rdeodoo) [inline addons/web_editor/controllers/main.py:812]: It would have been nice to let the user know about the limit (like `390 requests / 24h`), but let's not overthink this, it's not supposed to be something regular users will see, it's supposed to protect against attackers

**Review by rdeodoo — APPROVED** [review]
I'll merge it once IAP PR is merged and deployed, otherwise it will crash if no `database_id`.

This is working as expected:
image
For DB running the old community code, they will see this while they don't pull the last code
image

Everything is working fine:
- Prompt/Chat side, see above
- Website Placeholders, it just go through without generating anything

**Comment by rdeodoo** [comment]
@robodoo r+

**Pull request closed without merging**
